platform/registry.go

```diff
@@ -34,74 +33,187 @@ import (
+		return &ethereum.Platform{CoinIndex: coin.WAN}
+	case coin.Tomochain().Handle:
+		return &ethereum.Platform{CoinIndex: coin.TOMO}
+	case coin.Thundertoken().Handle:
+		return &ethereum.Platform{CoinIndex: coin.TT}
+	case coin.Cosmos().Handle:
+		return &cosmos.Platform{CoinIndex: coin.ATOM}
+	case coin.Kava().Handle:
+		return &cosmos.Platform{CoinIndex: coin.KAVA}
+	case coin.Tezos().Handle:
+		return &tezos.Platform{}
+	case coin.Aion().Handle:
+		return &aion.Platform{}
+	case coin.Icon().Handle:
+		return &icon.Platform{}
+	case coin.Iotex().Handle:
+		return &iotex.Platform{}
+	case coin.Ontology().Handle:
+		return &ontology.Platform{}
+	case coin.Theta().Handle:
+		return &theta.Platform{}
+	case coin.Tron().Handle:
+		return &tron.Platform{}
+	case coin.Vechain().Handle:
+		return &vechain.Platform{}
+	case coin.Zilliqa().Handle:
+		return &zilliqa.Platform{}
+	case coin.Waves().Handle:
+		return &waves.Platform{}
+	case coin.Aeternity().Handle:
+		return &aeternity.Platform{}
+	case coin.Bitcoin().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.BTC}
+	case coin.Litecoin().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.LTC}
+	case coin.Bitcoincash().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.BCH}
+	case coin.Dash().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.DASH}
+	case coin.Doge().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.DOGE}
+	case coin.Zcash().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.ZEC}
+	case coin.Zcoin().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.XZC}
+	case coin.Viacoin().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.VIA}
+	case coin.Ravencoin().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.RVN}
+	case coin.Qtum().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.QTUM}
+	case coin.Groestlcoin().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.GRS}
+	case coin.Zelcash().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.ZEL}
+	case coin.Decred().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.DCR}
+	case coin.Digibyte().Handle:
+		return &bitcoin.Platform{CoinIndex: coin.DGB}
+	case coin.Nebulas().Handle:
+		return &nebulas.Platform{}
+	case coin.Fio().Handle:
+		return &fio.Platform{}
+	case coin.Algorand().Handle:
+		return &algorand.Platform{}
+	case coin.Nano().Handle:
+		return &nano.Platform{}
+	case coin.Harmony().Handle:
+		return &harmony.Platform{}
+	case coin.Kusama().Handle:
+		return &polkadot.Platform{CoinIndex: coin.KSM}
+	default:
+		return nil
+	}
+}
+
+func getActivePlatforms(handle string) []blockatlas.Platform {
+	logger.Info("Loaded with: ", logger.Params{"handle": handle})
+
+	platform := getPlatform(handle)
+	if platform != nil {
+		return []blockatlas.Platform{platform}
+	}
+	return allPlatformsList
+}
 
-// CollectionAPIs contain platforms which collections services
-var CollectionAPIs map[uint]blockatlas.CollectionAPI
+func Init(platformHandle string) {
+	platformList := getActivePlatforms(platformHandle)
```

Comment:
is there a way to initialization only 2 or 3 platforms in one application? like Ethereum one, Bitcoin another one and some others small coins another one